### ISSUE TITLE
NavBar and Pay Button Text Customisation

### ISSUE THREAD

**Issue by kelvinnmugendi** [nelstein/nravepay#12]
I love the way this package is heading, I was about to create an issue for removing the 'Save Card' feature but that has been handled. Thank You
This is a customization issue, I need to customize the navbar text to read 'Add Card' not 'Card payment', also for the 'Pay' button if it could read something else like 'Add card' too. I will explain to the user beforehand that I will need to make a tiny charge in order for their card to be saved. Also, the Pay text and the currency on the right seem a bit off, maybe reduce the spacing between the two texts, this is in the pay button.
In summary,
1. Ability to customize the text on NavBar
2. Ability to customize the text on the button
3. Ability to remove the amount to be charged from that button when say, I have 'Add Card' text on the button. I don't want it to read 'Add Card Ksh 100'

**Comment by nelsonweze** [comment]
Yeah, I agree the package is too rigid now. My plan in the next couple of days is to make it more customizable. Will update this thread when it's done. PRs are welcome by the way.

**Comment by nelsonweze** [comment]
@kevinrobert3  Check if v1.0.6 solves this issue.

**Comment by nelsonweze** [comment]
Closing this issue. Reopen if not fixed
